reference/locations.v3.yml

```diff
@@ -1731,6 +1735,38 @@ components:
         enum:
           - asc
           - desc
+    date_created_min:
+      name: 'date_created:min'
+      description: |
+        'Query parameter that lets you filter by the minimum date created, for example, `2024-05-14T09:34:00Z`. Returns metafields created after this date.'
+      required: false
+      in: query
+      schema:
+        type: string
+    date_created_max:
+      name: 'date_created:max'
+      description: |
+        'Query parameter that lets you filter by the maximum date created, for example, `2024-05-14T09:34:00Z`. Returns metafields created before this date.'
```

Comment:
As previously I would state that `Query parameter that lets you filter by the date created which is before provided one`
FYI: it works similarly to `<=`